goodtablesio/plugins/github/templates/github_home.html

```diff
@@ -0,0 +1,50 @@
+{% extends "layout.html" %}
+
+{% block contents %}
+<div class="container">
+
+  <h1>GitHub</h1>
+
+  <p>GitHub integration description</p>
+
+  <h2>Repos</h2>
+
+  {% if current_user.is_authenticated %}
+
+    {% if sync %}
+
+      <button disable class="btn btn-warning">Syncing account</button>
+      <script>
+        window.location.reload(5);
```

Comment:
Oddly enough but it makes the trick) Our page seems loading something while repos are syncing.